blog-guidelines.md

```diff
@@ -0,0 +1,44 @@
+# ebpf.io Blog Guidelines #
+
+## ebpf.io Blog Overview ##
+
+The ebpf.io (CNCF) blog serves as a channel for eBPF project maintainers, contributors, and anyone interested in eBPF to share content with the eBPF community.
```

Comment:
Why the `(CNCF)`? Might be bit confusing since it's not a CNCF project.

Reply:
Otherwise lgtm.

Reply:
Seems I missed that part in the copy paste. I've removed now